extensions/amp-vimeo/validator-amp-vimeo.protoascii

```diff
@@ -14,11 +14,26 @@
 # limitations under the license.
 #
 
+tags: {  # amp-vimeo 1.0
+  html_format: AMP
+  tag_name: "SCRIPT"
+  satisfies: "amp-vimeo 1.0"
```

Comment:
could you add a test for the satisfies/excludes?

Reply:
How would that work?

Reply:
I'd imagine it would be one with both script tags, though it may issue a different error. What is being achieved by having satisifes/excludes here?

Reply:
Two script tags should throw the error for this satisfies/excludes.  For context the satisfies/excludes was added from a somewhat recent change to validator here: https://critique-ng.corp.google.com/cl/366894452

This allows us to specify different extension_spec rules between different versions of the script.  So in this case `1.0` does not have the following:

```
requires_usage: EXEMPTED
deprecated_allow_duplicates: true
```

Reply:
I can't get the following to fail, in either order:

```html
<script async custom-element="amp-vimeo" src="https://cdn.ampproject.org/v0/amp-vimeo-1.0.js"></script>
<script async custom-element="amp-vimeo" src="https://cdn.ampproject.org/v0/amp-vimeo-0.1.js"></script>
```

Could there be a problem with the definition that prevents this from erroring out?

Reply:
The definition looks right, I also tried producing locally and also not able to get it to fail.  I will check with the Validator team.

Reply:
Thanks Kevin. As discussed offline, the error is not being triggered in any case. Since that's being handled separately, I'll merge as-is.